lib/workers/branch/get-updated.ts

```diff
@@ -7,7 +7,7 @@ import { UpdateArtifactsConfig, ArtifactError } from '../../manager/common';
 
 export interface PackageFilesResult {
   artifactErrors: ArtifactError[];
-  parentBranch: string;
```

Comment:
Did we make it optional only because of tests?

Reply:
yes

Reply:
no, if you look at the function, it can be undefined

https://github.com/renovatebot/renovate/blob/c6f6ae7a8e91ba763b8f1a812508c8ea3da9de83/lib/workers/branch/get-updated.ts#L65-L67